src/styles/getTooltipStyles.js

```diff
@@ -0,0 +1,112 @@
+import {Dimensions} from 'react-native';
+import spacing from './utilities/spacing';
+import styles from './styles';
+import themeColors from './themes/default';
+import fontFamily from './fontFamily';
+import variables from './variables';
+
+const GUTTER_WIDTH = 16;
+
+/**
+ * The Expensify.cash repo is very consistent about doing spacing in multiples of 4.
+ * In an effort to maintain that consistency, we'll make sure that any distance we're shifting the tooltip
+ * is a multiple of 4.
+ *
+ * @param {Number} n
+ * @returns {Number}
+ */
+function roundToNearestMultipleOfFour(n) {
+    if (n > 0) {
+        return Math.ceil(n / 4.0) * 4;
+    }
+
+    if (n < 0) {
+        return Math.floor(n / 4.0) * 4;
+    }
+
+    return 0;
+}
+
+export default function getTooltipStyles(
+    interpolatedSize,
+    xOffset,
+    yOffset,
+    width,
+    height,
+    tooltipWidth,
+    tooltipHeight,
+) {
+    const windowWidth = Dimensions.get('window').width;
+    const shouldShowBelow = (yOffset - tooltipHeight) < GUTTER_WIDTH;
+    let horizontalShift = 0;
+    const tooltipLeftEdge = (xOffset + (width / 2)) - (tooltipWidth / 2);
+    const tooltipRightEdge = (xOffset + (width / 2)) + (tooltipWidth / 2);
+    if (tooltipLeftEdge < GUTTER_WIDTH) {
+        // Tooltip is in left gutter, shift right
+        horizontalShift = GUTTER_WIDTH - tooltipLeftEdge;
+    } else if (tooltipRightEdge > (windowWidth - GUTTER_WIDTH)) {
+        // Tooltip is in right gutter, shift left
+        horizontalShift = (windowWidth - GUTTER_WIDTH) - tooltipRightEdge;
+    }
+    horizontalShift = roundToNearestMultipleOfFour(horizontalShift);
+    return {
+        animationStyle: {
+            transform: [{
+                scale: interpolatedSize,
+            }],
+        },
+        tooltipWrapperStyle: {
+            position: 'absolute',
+            backgroundColor: themeColors.heading,
+            borderRadius: 4,
```

Comment:
We should have a global variable for this.

Reply:
Or rather, we could create one for this? Maybe like `componentBorderRadiusSmall: 4`